src/components/Admin/InvitationStatusBadge.tsx

```
@@ -0,0 +1,115 @@
+          className: 'text-green-600 border-green-600',
+          icon: CheckCircle,
+          text: 'Approved'
+        };
+
+      case 'notified':
+        const baseConfig = {
+          variant: 'outline' as const,
+          className: needsAttention ? 'text-orange-600 border-orange-600' : 'text-blue-600 border-blue-600',
+          icon: needsAttention ? AlertTriangle : MailCheck,
+          text: substage === 'expiring' ? 'Token Expiring' : 'Email Sent'
+        };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)